test/cpp/util/cli_credentials.cc

```diff
@@ -23,16 +23,20 @@
 DEFINE_bool(
     enable_ssl, false,
     "Whether to use ssl/tls. Deprecated. Use --channel_creds_type=ssl.");
-DEFINE_bool(use_auth, false, "Whether to create default google credentials.");
+DEFINE_bool(use_auth, false,
+            "Whether to create default google credentials. Deprecated. Use "
+            "--channel_creds_type=google_default_credentials.");
 DEFINE_string(
     access_token, "",
     "The access token that will be sent to the server to authenticate RPCs.");
 DEFINE_string(
     ssl_target, "",
     "If not empty, treat the server host name as this for ssl/tls certificate "
     "validation.");
-DEFINE_string(channel_creds_type, "",
-              "The channel creds type: insecure, ssl, or alts.");
+DEFINE_string(
+    channel_creds_type, "",
+    "The channel creds type: insecure, ssl, google_default_credentials or "
```

Comment:
nit: google_default_credentials is a bit long for a flag value :)

Reply:
gdc too pithy?

Reply:
PTAL